src/api/product/product.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { Product, ProductSelect } from './model';
+import { ProductArgs, ProductCreateInput } from './dto';
+import { PrismaService } from 'src/shared/datasource/prisma/prisma.service';
```

Comment:
same here, you can take it from 'src/shared/datasource' if im not wrong or maybe 'src/shared/datasource/prisma'